packages/scheduler/src/Scheduler.ts

```diff
@@ -73,15 +77,17 @@ export default class Scheduler {
     }
   }
 
-  _scheduleNextArrival (nextArrival) {
+  _scheduleNextArrival (nextArrival: Time): void {
     this._nextArrival = nextArrival
     const delay = Math.max(0, nextArrival - this.currentTime())
     this._timer = this.timer.setTimer(this._runReadyTasksBound, delay)
   }
 
-  _runReadyTasks () {
+  _runReadyTasks (): void {
     this._timer = null
     this.timeline.runTasks(this.currentTime(), runTask)
     this._scheduleNextRun()
   }
+
+  private _runReadyTasksBound = () => this._runReadyTasks()
```

Comment:
Nice.

Since this is basically an instance property (and not a method), how about we move it to the top with the other properties?